lang/pt_BR.json

```
@@ -42,7 +42,7 @@
     "Blog": "Blog",
     "Bounce Rate": "Taxa de Rejeição",
     "Brief": "Sumário",
-    "Bringing the games from the early generation of Pokémon games to the modern era.": "Trazendo os jogos das antigas gerações de Pokémon para a era moderna.",
+    "Bringing the games from the early generation of Pokémon games to the modern era.": "Trazendo os jogos das antigas gerações de Pokémon para a era moderna!",
```

```suggestion
    "Bringing the games from the early generation of Pokémon games to the modern era.": "Trazendo os jogos das antigas gerações de Pokémon para a era moderna.",
```
The period should not be replaced with exclamation mark